### PR TITLE
Cb 4 team

### PR DESCRIPTION
**Pull request by ImaadAziz**
* **Please check if the PR fulfills these requirements**
- [ ] The pull request title contains the ticket number in the title. For example [CB-1]
CB-4-Team

* **What kind of change does this PR introduce?** (Bug fix, new feature...)
Adds the team component.

* **What is the current behavior?What does the code do before you made the new changes?** 
There was no existing team component.

* **What is the new behavior?**
Adds the new component.

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR? Run a new command?)
No it should not.

* **Other information**:

### CODE REVIEW

File: frontend/src/App.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import logo from './logo.svg';
 import './App.css';
-import LandingPage from './components/LandingPage/LandingPage'
+import LandingPage from './components/LandingPage/LandingPage.js'
```
Comment (Codubee) [inline frontend/src/App.js]: The .js is implied so it does not have to be added to the import.

File: frontend/src/components/LandingPage/Team/Team.js [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+          Github link below."
+          link='https://github.com/Salmaan21'
+
+          />
+          
+      </div>
+    );
+};
+
+const TeamCard = (props) => {
```
Comment (Codubee) [inline frontend/src/components/LandingPage/Team/Team.js]: I would prefer to make the files have one component only. What do you think about creating another file containing the team card and then importing it into here.

**Review by Codubee — CHANGES_REQUESTED** [review]
Hi Imaad, great work! Left some comments to discuss some things.

File: frontend/src/components/LandingPage/Team/Team.js [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
```
Comment (jadendick) [inline frontend/src/components/LandingPage/Team/Team.js]: I think you need to delete this since you defined TeamCard in TeamCard.js

**Review by jadendick — CHANGES_REQUESTED** [review]

**Review by Codubee — APPROVED** [review]
LGTM!

**Pull request merged**